OpenRA.Mods.Common/Traits/SpawnActorsOnSell.cs

```diff
@@ -80,7 +81,7 @@ void Emit(Actor self)
 			if (eligibleLocations.Count == 0)
 				return;
 
-			var guaranteedActorTypes = Info.GuaranteedActorTypes?.Select(a =>
+			var guaranteedActorTypes = Info.GuaranteedActorTypes.Select(a =>
```

Comment:
This means `guaranteedActorTypes` can't be null below any more.

Reply:
it's a readonly field that is assigned an array on initialisation. How could it be null?

Reply:
There is a null check below that is redundant now...

Reply:
fixed